Documentation/guides.xml

```diff
@@ -18,4 +18,6 @@
     <!-- ext_sys_note and ext_sys_note_stable should always link to main-->
     <inventory id="ext_sys_note" url="https://docs.typo3.org/m/typo3/reference-coreapi/main/en-us/"/>
     <inventory id="ext_sys_note_stable" url="https://docs.typo3.org/m/typo3/reference-coreapi/main/en-us/"/>
+    <!-- explicitly link to version 11.3 of the news manual -->
+    <inventory id="ext/georgringer/news-11.3" url="https://docs.typo3.org/p/georgringer/news/11.3/en-us/"/>
```

Comment:
See above, here we use `ext/` as prefix, for sys_note we use `ext_`. We should not confuse users.

Reply:
for sys_note we also don't use the composer name. I would suggest to phase out the old keys, deprecate them after a while and then switch to composer names for each and every extension, including core extensions?

Reply:
See above

Reply:
As it is just an id, which references an URL and can be any string, I don't really care about that. But it should be consistent throughout the documentation (not only then guides.xml also the docs). There is no need for deprecation, as it has no impact for the user, If you name it "a" or "b/a". It is just a suggestion or convention.

Reply:
see above